packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -16,18 +16,21 @@ contract Staker is StakerInterface, Ownable, Testable {
      ****************************************/
 
     uint256 public emissionRate;
-    uint256 public cumulativeStaked;
+    uint256 public cumulativeActiveStake;
+    uint256 public cumulativePendingUnstake;
```

Comment:
Any reason we need to track this?

Reply:
it might not be needed...will double check.

Reply:
not needed and removed.